taworks/taform/views.py

```diff
@@ -230,34 +230,37 @@ def apply(request):
             else:
                 context = {
                     's_form' : s_form,
-                    'courses' : models.Course.objects.all(),
+                    'courses' : models.Course.objects.all().order_by('course_subject', 'course_id', 'course__section'),
                     'app_form' : a_forms,
                     'front_matter' : front_matter,
                     'AC' : AC,
                     'app_status' : app_status,
                     'status_date': status_date
                     }
                 return render(request, 'taform/application.html', context)
-            studentID=str(request.POST['student_id'])
-            apps_made= models.Application.objects.filter(student__student_id=studentID, preference__in = [1,2,3]).count()
-            app_id = 0
-            if apps_made > 0:
+            
+            studentID = str(request.POST['student_id'])
+            all_apps = models.Application.objects.all().filter(student__student_id=studentID).count()
```

Comment:
we should just pull from the student table.

if count with this studentId > 1 then that means duplicate submissions has happened.